simplivity/resources/virtual_machines.py

```diff
@@ -340,3 +340,20 @@ def power_off(self, timeout=-1):
         self.__refresh()
 
         return self
+
+    def power_on(self, timeout=-1):
+        """Power on virtual machine.
+
+        Args:
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            self: Returns the same object.
+        """
+        method_url = "{}/{}/power_on".format(URL, self.data["id"])
+        custom_headers = {'Content-type': 'application/vnd.simplivity.v1.14+json'}
+
+        self._client.do_post(method_url, None, timeout, custom_headers)
+        self.__refresh()
```

Comment:
FYI, refresh() is currently broken and reported in issue #36

Reply:
I have fixed the __refresh() in this PR.